pkg/controller/ipam/validate_test.go

```diff
@@ -152,7 +176,7 @@ func TestEgressControllerValidateExternalIPPool(t *testing.T) {
 						},
 						SubnetInfo: crdv1alpha2.SubnetInfo{
 							Gateway:      "10:2400::01",
-							PrefixLength: 64,
```

Comment:
why changing this? the new prefix /24 seems more unrealistic in ipv6.

Reply:
I added a check to prefix length must <32 for IPv4, so this case will fail for that reason (which is covered by a earlier test case already) if the length is still 64.

Reply:
got it, thanks for explanation.